ols/app/endpoints/ols.py

```diff
@@ -132,7 +98,7 @@ def conversation_request(
     else:
         summarizer_response = generate_response(
             conversation_id, llm_request, previous_input
-        )
+        )  # type: ignore[assignment]
```

Comment:
do we need those "ignores"? Is it because of Mypy?

Reply:
yes